rxlifecycle/src/main/java/com/trello/rxlifecycle/UntilCorrespondingEventObservableTransformer.java

```diff
@@ -30,11 +30,13 @@ public UntilCorrespondingEventObservableTransformer(@NonNull Observable<R> share
         return source.takeUntil(takeUntilCorrespondingEvent(sharedLifecycle, correspondingEvents));
     }
 
+    @NonNull
```

Comment:
True but in Android Studio I got a warning from an inspection telling me that also implementation classes should annotate the methods. If you want I can revert those changes too.